actionview/lib/action_view/helpers/text_helper.rb

```diff
@@ -250,12 +250,24 @@ def pluralize(count, singular, plural = nil, locale: nil)
       #
       #   word_wrap('Once upon a time', line_width: 1)
       #   # => Once\nupon\na\ntime
+      #
+      #   You can also specify a custom +break_sequence+ ("\n" by default)
+      #
+      #   word_wrap('Once upon a time', line_width: 1, break_sequence: "\r\n")
+      #   # => Once\r\nupon\r\na\r\ntime
+      #
+      #   You can also specify a custom +line_splitter+ if you want to have a special character sequence that defines a new line ("\n" by default)
+      #
+      #   word_wrap('Once upon a time,| in a kingdom called Far Far Away,| a king fell ill...', line_width: 10, line_splitter: "|")
+      #   # => "Once upon\na time,\nin a\nkingdom\ncalled Far\nFar Away,\na king\nfell\nill..."
       def word_wrap(text, options = {})
```

Comment:
When we're changing existing code it's okay to convert to keyword arguments like this:

```ruby
def word_wrap(text, line_with: 80, break_sequence: "\n")
```